src/main/java/at/hannibal2/skyhanni/config/features/garden/visitor/RewardWarningConfig.java

```diff
@@ -106,6 +106,12 @@ public class RewardWarningConfig {
     @FeatureToggle
     public boolean preventRefusingNew = true;
 
+    @Expose
+    @ConfigOption(name = "Block Accepting Without Rabbit", desc = "Prevent accepting a visitor without a Rabbit pet equipped. Useful for people who can't afford autopet rules.")
```

Comment:
A slightly better description might be 
```suggestion
    @ConfigOption(name = "Block Accepting Without Rabbit", desc = "Prevent accepting a visitor without a Rabbit pet equipped. Useful if you can't afford autopet rules.")
```